cdap-app-fabric/src/main/java/co/cask/cdap/app/guice/DistributedProgramRunnableModule.java

```diff
@@ -113,6 +115,7 @@ protected void configure() {
             .in(Scopes.SINGLETON);
           // bind AuthorizationEnforcer to AuthorizationEnforcementService
           bind(AuthorizationEnforcer.class).to(AuthorizationEnforcementService.class).in(Scopes.SINGLETON);
```

Comment:
no, this is still needed. this is where the ``enforce`` method would be called from